src/main.py

```diff
@@ -11,6 +12,9 @@ def main(cli_options: dict) -> None:
 
     blocks.log_starting_messages(logger, init_params, cli_options)
 
+    config = MergedConf(logger, cli_options)
+    print(config.final)
```

Comment:
We will add nice output via outputs.blocks.Blocks later on. For now, please use `logger.debug`. Since logging module is configured in this project, we should refrain from print() statements as much as possible.